prql-compiler/src/semantic/resolver.rs

```diff
@@ -650,7 +685,7 @@ mod tests {
         let result = parse(prql).and_then(resolve_and_translate).unwrap();
         assert_snapshot!(result, @r###"
         SELECT
-          first_name OVER (PARTITION BY last_name) AS first_name
+          first_name
```

Comment:
Is this still a correct translation?

Reply:
In this case `first_name` is not a window function. It's a scalar function (identity actually), so it applies to a single row at a time, which does not depend on groups, sorting or window frames.